sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerModelFactory.cs

```diff
@@ -442,16 +504,22 @@ public static RecognizedForm RecognizedForm(string formType, FormPageRange pageR
             new RecognizedFormCollection(list);
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="FormRecognizer.Models.FormSelectionMark"/> class.
+        /// Initializes a new instance of the <see cref="FormRecognizer.Models.TextAppearance"/> class.
         /// </summary>
-        /// <param name="boundingBox">The quadrilateral bounding box that outlines the element.</param>
-        /// <param name="pageNumber">The 1-based number of the page in which this element is present.</param>
-        /// <param name="text">The text of selection mark value.</param>
+        /// <param name="style">An object representing the style of the text line.</param>
+        /// <returns>A new <see cref="FormRecognizer.Models.TextAppearance"/> instance for mocking.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="style"/> is null.</exception>
+        public static TextAppearance TextAppearance(TextStyle style) =>
+            new TextAppearance(style);
```

Comment:
This one throws an `ArgumentNullException` because that's what the generated constructor does by default. Other model factory methods don't have this check. I think it may be worth to add an extra internal constructor to keep the behavior the same as in other methods.